stdlib/public/Darwin/Foundation/NSError.swift

```diff
@@ -235,7 +235,7 @@ public func _getErrorDefaultUserInfo<T: Error>(_ error: T)
       _errorDomainUserInfoProviderQueue.sync {
         if NSError.userInfoValueProvider(forDomain: domain) != nil { return }
         NSError.setUserInfoValueProvider(forDomain: domain) { (nsError, key) in
-          let error = nsError as Error
+          let error = nsError 
```

Comment:
If we aren't performing this cast, we don't need a distinct `error` and `nsError` here, right? Cleanest to probably just rename the parameters in the line above.

Reply:
Right, Thank's @natecook1000 :)